public/app/plugins/datasource/grafana-azure-monitor-datasource/utils/migrateQuery.test.ts

```diff
@@ -98,112 +67,18 @@ const modernMetricsQuery: AzureMonitorQuery = {
 
 describe('AzureMonitor: migrateQuery', () => {
   it('modern queries should not change', () => {
-    const result = migrateQuery(modernMetricsQuery, templateSrv, setErrorMock);
+    const result = migrateQuery(modernMetricsQuery);
 
     // MUST use .toBe because we want to assert that the identity of unmigrated queries remains the same
     expect(modernMetricsQuery).toBe(result);
   });
 
-  describe('migrating from a v7 query to the latest query version', () => {
```

Comment:
Curiosity question: Is that removed because we stopped supporting version under 8.

Reply:
nope, I have removed this the format in v7 is the same than in v9.1 (at least for the properties that this test want meant to verify).